Amazon.QLDB.Driver/session/QldbSession.cs

```diff
@@ -44,12 +44,10 @@ namespace Amazon.QLDB.Driver
     /// </list>
     /// </para>
     /// </summary>
-    internal class QldbSession
+    internal class QldbSession : BaseQldbSession
     {
         private readonly ILogger logger;
```

Comment:
This can probably be shared in the base.

Reply:
Since the logger is readonly, removing it from QldbSession and AsyncQldbSession results in the constructors complaining "A readonly field cannot be assigned to (except in a constructor or init-only setter of the type in which the field is defined or a variable initializer)".

Reply:
Then add it as a part of the abstract base constructor.

Reply:
Done.